### PR TITLE
added placeholder with example comment for deployment comment

### PR DESCRIPTION
**Pull request by maciej-brzezinski**

### CODE REVIEW

File: ui/client/components/CommentInput.tsx [hunk 1]
```diff
@@ -4,13 +4,13 @@ import {TextAreaWithFocus} from "./withFocus"
...
 
-export const CommentInput = ({onChange, value, ...props}: Props): JSX.Element => {
+export const CommentInput = ({onChange, value, defaultValue, ...props}: Props): JSX.Element => {
   const {t} = useTranslation()
   return (
     <TextAreaWithFocus
       {...props}
       value={value || ""}
-      placeholder={t("commentInput.placeholder", "Write a comment...")}
+      placeholder={defaultValue?.toString() || t("commentInput.placeholder", "Write a comment...")}
```
Comment (lciolecki) [inline ui/client/components/CommentInput.tsx:13]: It's already string.

File: ui/client/actions/nk/assignSettings.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ export type FeaturesSettings = {
   remoteEnvironment: { targetEnvironmentId: string },
   environmentAlert: { content: string, cssClass: string },
   commentSettings: { substitutionPattern: string, substitutionLink: string },
+  deploymentCommentSettings: { validationPattern: string, exampleComment: string },
```
Comment (lciolecki) [inline ui/client/actions/nk/assignSettings.ts:18]: `validationPattern` is not used.

File: ui/client/components/CommentInput.tsx (hunk 1 above)
Comment (maciej-brzezinski) [reply ui/client/components/CommentInput.tsx:13]: for this component defaultValue is of type 'string | number | readonly string[]' so I have to specify it like that

Comment (lciolecki) [reply ui/client/components/CommentInput.tsx:13]: Ahh, now I understood ;)

**Review by arkadius — APPROVED** [review]

**Pull request merged**